api/src/main/scala/hmda/api/HmdaApi.scala

```diff
@@ -36,9 +41,27 @@ object HmdaApi
 
   //Start up API Actors
 
-  createSingleLarValidator(system)
+  val supervisor = HmdaSupervisor.createSupervisor(system)
+
+  (supervisor ? FindActorByName(SingleLarValidation.name))
```

Comment:
Yes, as I stated in my previous comment, the logic for self shutdown is only present in persistent actors. We will have thousands of them, hence this logic. By the way, this will change when we go to a clustered application, but I don't want to get ahead of myself. Also as I said in my reply to @nickgrippin, I'm making some changes to the way `InstitutionPersistence` is being instantiated. Will push additional commits soon